app/talents/my-profile/education/page.tsx

```diff
@@ -0,0 +1,253 @@
+"use client";
+
+import { useState, FormEvent } from "react";
+
+import toast from "react-hot-toast";
+
+import { SelectInput } from "../../../components/select-input";
+import { degrees } from "../../../constants/degrees";
+
+interface Option {
+  value: string;
+  label: string;
+}
+export default function CreateJob() {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  const [selectedDegree, setSelectedDegree] = useState<Option | null>(null);
+  const [selectedStartMonth, setSelectedStartMonth] = useState<Option | null>(
+    null
+  );
+  const [selectedStartYear, setSelectedStartYear] = useState<Option | null>(
+    null
+  );
+  const [selectedEndMonth, setSelectedEndMonth] = useState<Option | null>(null);
+  const [selectedEndYear, setSelectedEndYear] = useState<Option | null>(null);
+
+  const handleSubmit = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setIsLoading(true);
+
+    const formData = new FormData(e.currentTarget);
+
+    const startYear = Number(selectedStartYear?.value);
+    const startMonth = Number(selectedStartMonth?.value);
+    const startDate = new Date(startYear, startMonth);
+
+    const endYear = Number(selectedEndYear?.value);
+    const endMonth = Number(selectedEndMonth?.value);
+    const endDate = new Date(endYear, endMonth);
+
+    const dataForm = {
```

Comment:
It's a generic name. `educationForm` is more explicit.

It's more useful if we have multiple form. It's not the case but it's better to anticipate this.